docs/blog/0.41.md

```diff
@@ -0,0 +1,56 @@
+---
+
+date: 2024-02-22
+image: https://github.com/rilldata/rill/assets/5587788/b30486f6-002a-445d-8a1b-955b6ec0066d
+
+---
+
+# Rill 0.41 - Pivot tables, more defaults, picking up where you left off and more! 🚀
+
+:::note
+⚡ Rill Developer is a tool that makes it effortless to transform your datasets with SQL and create fast, exploratory dashboards.
+
+To [try out Rill Developer, check out these instructions](../../install) and [join us over on Discord](https://bit.ly/3bbcSl9) to meet the team behind the product as well as other users!
+:::
+
+![release-0 41](https://storage.googleapis.com/prod-cdn.rilldata.com/docs/release-notes/release-0.41-gif)
+
```

Comment:
High level items to add
- AI powered Dashboard generation 
- Clickhouse live connection support